app/assets/javascripts/datapicker.js

```diff
@@ -0,0 +1,6 @@
+$( document ).on('turbolinks:load', function() {
+    $('.datepicker').datepicker({
+        todayHighlight: true,
+        format: "dd-mm-yyyy"
```

Comment:
Please add ``` autoclose: true ```. It close the calendar after a value was choiced.

Reply:
I see. Format is wrong :(